Packs/ContextualThreatScoring/README.md

```diff
@@ -0,0 +1,105 @@
+This pack lets you calculate custom threat scores for indicators of compromise (IOC(s)).
+
+In Cortex XSOAR, an indicator’s verdict is assigned according to the verdict returned by the source with the highest reliability. For more information, check our documentation on [Indicator Verdict](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.5/Cortex-XSOAR-Threat-Intel-Management-Guide/Indicator-Verdict).
+
+To increase the efficacy of the threat data collected across multiple sources, we need to look at more parameters beyond an indicator’s verdict. For example, it is common knowledge that signatures like IP addresses and hashes are not used by an attacker long after being discovered. So, we could use an indicator’s age and de-prioritize indicators that are older.
+
+#### Design
+
+In this version, the custom scores are derived from certain pre-defined factors in a list.
+1. Factor values can be of different types.
+	1. Continuous: Values of a factor can be in a range, like *relatedIncidentCount*. This is true for factors that have a numerical format.
+	2. Discrete: Values of a factor can be discrete and mapped to individual keys. For example, *geocountry* value can be *US*.
+1. Factor values can be obtained from an indicator in different ways.
+	1. Derived: It can be derived from a field. For example, *age* can be calculated from indicator field *firstSeen*. There are custom calculations required for derived factors, and so these cannot be added dynamically to the list of factors. The derived factors are:
+		1. *age*: An indicator’s age is calculated as the time difference between *now* and when the indicator was first seen.
+		2. *weightedScore*: An indicator’s weighted score is calculated as by taking each source’s reliability as weight and associated verdict as value.
+			$$weightedScore = \frac{\sum_{i=0}^{N}{{source\_reliability}^2_i.score_i}}{\sum_{i=0}^{N}{source\_reliability_i}}$$
+	1. Directly extracted: It can be extracted from an existing field. For example, *geocountry* can be extracted from an existing indicator field directly.
```

Comment:
```suggestion
	2. Directly extracted: It can be extracted from an existing field. For example, *geocountry* can be extracted from an existing indicator field directly.
```